pkgs/development/python-modules/pytest-image-diff/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, buildPythonPackage
+, fetchFromGitHub
+, typing-extensions
+, pytest
+, diffimg
+, imgdiff
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-image-diff";
+  version = "0.0.11";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "Apkawa";
+    repo = "pytest-image-diff";
+    rev = "v${version}";
+    hash = "sha256-7GBwxm0YQNN/Gq1yyBIxCEYbM7hmOFa9kUsfbBKQtBQ=";
+  };
+
+  propagatedBuildInputs = [
+    pytest
```

```suggestion
  buildInputs = [
    pytest
  ];

  propagatedBuildInputs = [
```
this is special for pytest otherwise overwriting it is really hard

---

doh, right, it shouldn't be propagated
switched it to nativeCheckInputs instead of buildInputs though, i hope that's ok

---

woeps, seems like propagating it was a good idea after all, KiCad's test phase needs it xD

---

We don't propagate it even if it would be correct. The consume should bring its own pytest or better pytestCheckHook